tests/e2e/e2e.sh

```diff
@@ -17,83 +17,57 @@
-  if ! (kind delete cluster --name=istio-testing) > /dev/null; then
-    echo "No existing kind cluster with name istio-testing. Continue..."
-  fi
-
-  trap cleanup_kind_cluster EXIT
-
-  # Create KinD cluster
-  if ! (kind create cluster --name=istio-testing --loglevel debug --retain --wait 30s); then
-    echo "Could not setup KinD environment. Something wrong with KinD setup. Exporting logs."
-    exit 1
-  fi
-
-  KUBECONFIG="$(kind get kubeconfig-path --name="istio-testing")"
-  export KUBECONFIG
-
-cat <<EOF > metallb-config.yaml
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  namespace: metallb-system
-  name: config
-data:
-  config: |
-    address-pools:
-    - name: default
-      protocol: layer2
-      addresses:
-      - 172.17.255.1-172.17.255.250
-EOF
-
-  kubectl apply -f https://raw.githubusercontent.com/google/metallb/v0.8.1/manifests/metallb.yaml
-  kubectl apply -f ./metallb-config.yaml
-}
 
 function setup_docker() {
-  HUB=istio-testing TAG=latest make -f Makefile.core.mk controller docker
-  kind --loglevel debug --name istio-testing load docker-image istio-testing/operator:istio-testing
+  HUB=istio-testing TAG=1.5-dev make controller docker
+  kind --loglevel debug --name istio-testing load docker-image istio-testing/operator:1.5-dev
+  kind --loglevel debug --name istio-testing load docker-image istio-testing/app:istio-testing
 }
 
-setup_kind_cluster
-setup_docker
+
 
 mkdir -p "${ARTIFACTS}/out"
 
-ISTIO_DIR="${GOPATH}/src/istio.io/istio"
+ISTIO_DIR="${ROOT}/src/istio.io/istio"
 
 # Create a clone of the Istio repository
 if [[ ! -d "${ISTIO_DIR}" ]]
 then
-  git clone https://github.com/sdake/istio.git "${ISTIO_DIR}"
+  git clone https://github.com/istio/istio.git "${ISTIO_DIR}"
 fi
 
 # Create an operator manifest from the default control plane configuration
-
-operator_manifest_files=( "deploy/namespace.yaml" "deploy/crds/istio_v1alpha2_istiocontrolplane_crd.yaml" "deploy/crds/istio_v1alpha2_istiocontrolplane_cr.yaml" "deploy/service_account.yaml" "deploy/clusterrole.yaml" "deploy/clusterrole_binding.yaml" "deploy/service.yaml" "deploy/operator.yaml" )
+cd "${ROOT}/src/istio.io/operator"
+operator_manifest_files=( "deploy/namespace.yaml" "deploy/crds/istio_v1alpha2_istiocontrolplane_crd.yaml" "deploy/service_account.yaml" "deploy/clusterrole.yaml" "deploy/clusterrole_binding.yaml" "deploy/service.yaml" "deploy/operator.yaml" "deploy/crds/istio_v1alpha2_istiocontrolplane_cr.yaml" )
 
 # Generate the main manifest
 rm -f "${ISTIO_DIR}"/install/kubernetes/istio-operator.yaml
 for manifest_file in "${operator_manifest_files[@]}"
 do
 	cat "${manifest_file}" >> "${ISTIO_DIR}"/install/kubernetes/istio-operator.yaml
-	echo "---" >> "${ISTIO_DIR}"/install/kubernetes/istio-operator.yaml
 done
 
-kubectl get pods --all-namespaces -o wide
+
+#kind cluster setup
+pushd "${ISTIO_DIR}"
+# shellcheck disable=SC1091
+source "./prow/lib.sh"
+setup_kind_cluster kindest/node:v1.15.3
```

Comment:
I don't think we should explicitly set the node here, or this will break when we update kind. Really, any of this can break if we update ./prow/lib.sh so its already sort of at risk

Reply:
right, my suggestion was to keep this per repo - and then follow this work up with a common-files replication.  agree re node, we should bake node into the buildtools image imo.

Reply:
removed the node images here